app/controllers/api/v3/courses_controller.rb

```diff
@@ -24,11 +28,21 @@ def show
 
     private
 
+      def allowed_to_disable_pagination?
```

Comment:
:confused:  
Is this used `allowed_to_disable_pagination?`

Reply:
See: https://github.com/DFE-Digital/teacher-training-api/blob/master/app/controllers/api/v3/application_controller.rb#L24-L28

Reply:
oh right, but still ,this doesn't make sense...
bad inheritance?


```
class CoursesController < API::V3::ApplicationController
class ProviderSuggestionsController < API::V3::ApplicationController
class ProvidersController < API::V3::ApplicationController
class SubjectAreasController < API::V3::ApplicationController
```

`CoursesController` is the only thing that uses it...

Reply:
Yes, it's now a feature for the v3 api so if someone wants to disable pagination for an endpoint, they are able to do so